tests/test_visitors/test_ast/test_conditions/test_negated_conditions.py

```diff
@@ -12,9 +12,18 @@
     ...
 """
 
+complex_elif_conditions = """
+if {0}:
+    ...
+elif ...:
+    ...
+"""
+
 complex_conditions = """
```

Comment:
Can you please leave the test case:

```python
if {0}:
    ...
else:
    ...
```

And create a new one:

```python
if {0}:
    ...
elif ...:
    ...
else:
    ...
```

Reply:
@sobolevn I've parameterized the tests with the template variable, since the behavior should not differ in this case from whether there is an `elif` block or not.